pyomo/contrib/viewer/tests/test_data_model_item.py

```diff
@@ -29,10 +29,46 @@ def blackbox(a,b):
             )
         m.c1 = Constraint(expr=m.x[0] * m.z[0]**2 + self.bb(m.x[0],m.x[1]) == 2*sqrt(2.0))
         m.c2 = Constraint(expr=m.z[2]**4 * m.z[1]**2 + m.z[1] == 8+sqrt(2.0))
-
+        m.c3 = Constraint(expr=m.x[1] == 3)
         self.m = m.clone()
 
     def test_expr_calc(self):
         cdi = ComponentDataItem(parent=None, ui_setup=None, o=self.m.b1.e1)
         cdi.calculate()
         assert(abs(cdi.get("value")-3) < 0.0001)
+
```

Comment:
We missed it the first time, but this file is also missing a copyright statement.